README.md

```
@@ -1 +1,241 @@
+tuist generate
+
+# 5. Xcode에서 NDGL-iOS.xcworkspace 열기
+```
+
+## 👥 팀원
+
+<table>
+  <tr>
+    <td align="center">
+      <a href="https://github.com/KimNahun">
+        <img src="https://github.com/KimNahun.png" width="120" />
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)